cmd/crossplane/core/init.go

```diff
@@ -72,6 +72,11 @@ func (c *initCommand) Run(s *runtime.Scheme, log logging.Logger) error {
 	}
 	steps = append(steps,
 		initializer.NewTLSCertificateGenerator(c.Namespace, c.TLSCASecretName, tlsGeneratorOpts...),
+		// Crossplane used to serve these webhooks, but now uses CEL validation.
+		initializer.NewValidatingWebhookRemover("crossplane",
```

Comment:
will this only do real work on the first upgrade from v1? or it is useful every time crossplane starts up? trying to understand the lifecycle of webhooks better now with these changes

Reply:
It'll only do real work on the first upgrade, yeah.

After that it'll look for the `crossplane` ValidatingWebhookConfiguration, find it doesn't exist, and return early.